sources/core/objects/Glass.cpp

```diff
@@ -3,8 +3,10 @@
 #include "../SyncRandom.h"
 #include "../Helpers.h"
 #include "Mob.h"
+#include "Breakable.h"
+#include "Shard.h"
 
-FlatGlass::FlatGlass(quint32 id) : Structure(id)
+FlatGlass::FlatGlass(quint32 id) : Breakable(id)
 {
     transparent = true;
```

Comment:
`FlatGlass` should have more hp than default 1. Something like 20 for an usual window and 100 for a reinforced window looks good.